data_safe_haven/commands/context.py

```diff
@@ -18,7 +19,11 @@
 @context_command_group.command()
 def show() -> None:
     """Show information about the selected context."""
-    settings = ContextSettings.from_file()
+    try:
+        settings = ContextSettings.from_file()
+    except DataSafeHavenConfigError:
+        print("No context selected.")
+        return
```

Comment:
We should add a test for the case where there is no context.

Reply:
fixed in a3d9eba2686240f73964dbafa13a14ddbaaa24f0 and fb280b9536e12fcd82bf9c3624f8f97f9ed619b4